plugins/flytekit-vaex/tests/test_vaex_plugin_sd.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import vaex
+from flytekitplugins.vaex.sd_transformers import VaexDataFrameRenderer
+from typing_extensions import Annotated
+
+from flytekit import kwtypes, task, workflow
+from flytekit.types.structured.structured_dataset import PARQUET, StructuredDataset
+
+full_schema = Annotated[StructuredDataset, kwtypes(x=int, y=str), PARQUET]
```

Comment:
Just an observation:
i initially assumed col types metadata was skipped then it would still be ok as still have two arguments to `Annotated`
https://github.com/flyteorg/flytekit/blob/b7ecdf60768bf8b03a849947d5e2cab0fccde1e3/plugins/flytekit-polars/tests/test_polars_plugin_sd.py#L10

but if i do  `full_schema = Annotated[StructuredDataset, PARQUET]`, i get the following error

<img width="1028" alt="Screenshot 2022-10-19 at 23 56 33" src="https://user-images.githubusercontent.com/16509490/196820825-3f424411-a7fa-4196-89c1-93f983849b1c.png">

Reply:
cc: @pingsutw

Reply:
hmm, I fetched your commit, and reran the test (has updated to Annotated[StructuredDataset, PARQUET]) but didn't get the error. Let's wait to see if ci can pass.

Reply:
> hmm, I fetched your commit, and reran the test (has updated to Annotated[StructuredDataset, PARQUET]) but didn't get the error. Let's wait to see if ci can pass.

thanks, test that has failed in ci is some other  one (unrelated to this PR)